### DISCUSSION TITLE
Disabling Tab Switcher

### DISCUSSION THREAD

**Discussion by franciscop** [zed-industries/zed#17091] [Feature Requests]
I like that `ctrl+tab` goes to the next tab, and `ctrl+shift+tab` to the previous one in the top. I don't like the popup "Tab Switcher". Is there a way to disable it? I've been trying things like `"tab_switcher": { "show": false }` in the settings and that didn't seem to work. Seeing [the original PR that added the Tab Switcher](https://github.com/zed-industries/zed/pull/7987/), this seems to work (since it basically reverts the key bindings), so I put it here for anyone interested and in case someone has a better suggestion:

```js
// For information on binding keys, see the Zed
// documentation: https://zed.dev/docs/key-bindings
[
  {
    "context": "Workspace",
    "bindings": {
      "ctrl-tab": "pane::ActivateNextItem",
      "ctrl-shift-tab": "pane::ActivatePreviousItem"
    }
  },
  // ...
]
```

Edit: edited as per the comment. I had it already edited in my files and it works fine, I guess it was automatically changed in some upgrade? Which was pretty cool

**Comment by ibnuh** [comment]
`pane::ActivatePrevItem` is now deprecated. Use the new `pane::ActivatePreviousItem` action:

```
[
  {
    "context": "Workspace",
    "bindings": {
      "ctrl-tab": "pane::ActivateNextItem",
      "ctrl-shift-tab": "pane::ActivatePreviousItem"
    }
  },
  // ...
]
```

**Reply by franciscop** [reply]
Thanks, updated the comment!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links

**Comment by franciscop** [comment]
I think these were wiped out after a recent update, but adding them again made it work as expected 👍
